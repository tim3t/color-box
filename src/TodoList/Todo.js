import React from 'react';

const Todo = () => {
	return (
		<div>
			<p>This is a todo item</p>
		</div>
	);
};

export default Todo;
