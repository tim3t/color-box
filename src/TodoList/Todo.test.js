import React from 'react';
import { render } from '@testing-library/react';
import Todo from './Todo';

it('should render without exploding the app', function() {
	render(<Todo />);
});

it('should match the snapshot test', function() {
	const { asFragment } = render(<Todo />);
	expect(asFragment).toMatchSnapshot();
});
