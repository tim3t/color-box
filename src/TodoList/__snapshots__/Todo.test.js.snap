// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot test 1`] = `[Function]`;
